websites/W/watchmovieshd/dist/metadata.json

```
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Toxicity",
+		"id": "794558195921846292"
+	},
+	"service": "watchmovieshd",
+	"description": {
+		"en": "This is a website where you can watch tv shows and movies for free"
```

Tried to make something of the meta description (watch movies hd online, watch movies hd free online, free watch movies hd, free watch movies hd online, update daily, hd quality)
```suggestion
		"en": "Watch movies that get updated daily online in hd quality."
```